kudu-handler/src/test/queries/positive/kudu_queries.q

```diff
@@ -0,0 +1,148 @@
+--! qt:dataset:src
```

Comment:
You do not seem to be using `src` table, you can remove this line so it is not loaded.